src/fhi_aims_workflows/schemas/task.py

```diff
@@ -185,6 +186,60 @@ def from_aims_calc_doc(cls, calc_doc: Calculation) -> "OutputSummary":
             trajectory=calc_doc.output.atomic_steps,
         )
 
+class ConvergenceSummary(BaseModel):
+    """Summary of the outputs for an FHI-aims convergence calculation."""
```

Comment:
Can you add a more complete docstring outlining what the parameters here as well?

Reply:
Actually for schema classes themselves we don't have docstrings with parameters (see eg [InputSummary](https://github.com/tpurcell90/atomate2-fhi-aims/blob/ebd9f9776dc0879cb84c1d66ceec579b911f2bc1/src/fhi_aims_workflows/schemas/task.py#L98), [OutputSummary](https://github.com/tpurcell90/atomate2-fhi-aims/blob/ebd9f9776dc0879cb84c1d66ceec579b911f2bc1/src/fhi_aims_workflows/schemas/task.py#L134C4-L134C4)). I guess that's because all the dataclass parameters have the description field, which should be self-explanatory. However, we need to look into the atomate2 policy on that and mimic it, I think

Reply:
They have auto documentation from docstrings, I will see what that looks like with the data classes